crates/turbopack-core/src/chunk/chunk_in_group.rs

```diff
@@ -0,0 +1,131 @@
+use anyhow::Result;
+use turbo_tasks::{primitives::StringVc, ValueToString, ValueToStringVc};
+use turbo_tasks_fs::FileSystemPathVc;
+
+use super::{Chunk, ChunkVc, ParallelChunkReferenceVc};
+use crate::{
+    asset::{Asset, AssetContentVc, AssetVc},
+    chunk::ParallelChunkReference,
+    introspect::{
+        asset::{children_from_asset_references, content_to_details, IntrospectableAssetVc},
+        Introspectable, IntrospectableChildrenVc, IntrospectableVc,
+    },
+    reference::AssetReferencesVc,
+    version::VersionedContentVc,
+};
+
+#[turbo_tasks::value]
+pub struct ChunkInGroup {
```

Comment:
We're going to need a comment on this struct to explain the difference between this and `Chunk`/`ChunkGroup`.